components/central-application-gateway/README.md

```diff
@@ -110,6 +110,12 @@ As a result, Central Application Gateway:
 A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CR must be unique for a given application.
 Invocation of endpoints with duplicate names results in a `400 Bad Request` failure. In such a case, you must change one of the names to avoid ambiguity.
 
+### Status codes for errors returned by Application Gateway
+
+- when application specified in path doesn't exist Application Gateway returns `404 Not Found`
+- when application, service or entry for Compass use case is not specified in the path, Application Gateway returns `400 Bad Request `
+- when call to target API timeouts Application Gateway returns `504 Gateway Timeout`
+
```

Comment:
```suggestion
### Status codes for errors returned by Application Gateway

- `404 Not Found` - returned when the Application specified in the path doesn't exist.
- `400 Bad Request` - returned when an Application, service, or entry for the [Compass mode](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/application-connectivity/) is not specified in the path.
- `504 Gateway Timeout` - returned when a call to the target API times out.
```

Alternatively, what if we approach it from the other side and start with the codes followed by their description? 
I think it might be a bit easier to navigate when debugging as well, just as in a dictionary, you don't look up words by definition.

Reply:
I like it!